README.md

```diff
@@ -1,5 +1,6 @@
 # ZSWatch Hardware
 
+[![License](https://img.shields.io/badge/License-GPL%203.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
Badge is GPL, but the URL links to Apache => update URL to https://opensource.org/license/gpl-3-0/
Same on the dock

Reply:
@jakkra it´s the same link as in the ZSWatch Repo README. So we have to change it there too